### ISSUE TITLE
Can't close window without ending in task manager

### ISSUE THREAD

**Issue by janus0** [fuhsjr00/bug.n#116]
Currently using windows 10 pro, and any time I copy/paste a file from one drive to another, the windows transfer dialog opens, but won't close once the transfer is complete. I have to use task manager to restart explorer.exe to get rid of the dialog. This problem is linked to my bug.n setup as it doesn't occur when bug.n is closed. I've had this happen with a select few other programs (can't remember which).

**Comment by joten** [comment]
You may set a "Config_rule" like `Config_rule_#11=OperationStatusWindow;.*;;0;0;0;1;1;0;` excluding the window from being managed. You will need the class name or/ and title to identify the window (retreivable with <kbd>Win</kbd><kbd>i</kbd>) as documented in [doc/Configuring_rules](https://github.com/fuhsjr00/bug.n/blob/master/doc/Configuring_rules.md).

**Comment by janus0** [comment]
Seems to be fixed. I'll open another ticket if it happens again and I can't resolve it. Thanks!
